Reduce the number of digits checked for kmmd_pmemd/kmmd_pmemd:gb tests.
This was also done for earlier easyconfigs for Amber, but it is not clear
if the Amber developers agree with this approach.

steven.vandenbrande@kuleuven.be
--- amber24_src/test/cuda/kmmd/kmmd_pmemd_gb/Run.kmmd.orig	2025-04-03 15:14:55.000884000 +0200
+++ amber24_src/test/cuda/kmmd/kmmd_pmemd_gb/Run.kmmd	2025-04-03 15:16:07.412693000 +0200
@@ -67,7 +67,7 @@
 #cp $output            $output.save
 
 ##diff the test and reference
-../../../dacdif -f $output.save $output 
+../../../dacdif -a 0.1 -f $output.save $output 
 
 /bin/rm -f mdin mdinfo mdcrd mdfrc restrt tidied*txt *.dat kmmd.std* logfile
 exit(0)
--- amber24_src/test/hip/kmmd/kmmd_pmemd_gb/Run.kmmd.orig	2025-04-03 15:16:50.516452000 +0200
+++ amber24_src/test/hip/kmmd/kmmd_pmemd_gb/Run.kmmd	2025-04-03 15:17:02.674848000 +0200
@@ -67,7 +67,7 @@
 #cp $output            $output.save
 
 ##diff the test and reference
-../../../dacdif -t 2  $output.save $output 
+../../../dacdif -a 0.1  $output.save $output 
 
 /bin/rm -f mdin mdinfo mdcrd mdfrc restrt tidied*txt *.dat kmmd.std* logfile
 exit(0)
--- amber24_src/test/kmmd/kmmd_pmemd_gb/Run.kmmd.orig	2025-04-04 13:47:38.029284000 +0200
+++ amber24_src/test/kmmd/kmmd_pmemd_gb/Run.kmmd	2025-04-04 13:48:03.964956000 +0200
@@ -64,7 +64,7 @@
 #cp $output            $output.save
 
 ##diff the test and reference
-../../dacdif -t 2  $output.save $output 
+../../dacdif -a 0.1  $output.save $output 
 
 /bin/rm -f mdin mdinfo mdcrd mdfrc restrt tidied*txt *.dat
 exit(0)
